src/helm/ralph/templates/deployment.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "ralph.fullname" . }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    {{- include "ralph.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "ralph.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "ralph.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      securityContext:
+        {{- toYaml .Values.securityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            {{- toYaml .Values.ralph_lrs.command | nindent 12 }}
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+          livenessProbe:
+            httpGet:
+              path: /__heartbeat__
+              port: {{ .Values.ralph_lrs.port }}
+              httpHeaders:
+                - name: Host
+                  value: "lrs.{{ .Values.namespace }}.{{ .Values.ralph_lrs.domain_name }}"
+            initialDelaySeconds: 15
+            periodSeconds: 30
+          readinessProbe:
+            httpGet:
+              path: /__lbheartbeat__
+              port: {{ .Values.ralph_lrs.port }}
+              httpHeaders:
+                - name: Host
+                  value: "lrs.{{ .Values.namespace }}.{{ .Values.ralph_lrs.domain_name }}"
+            initialDelaySeconds: 5
+            periodSeconds: 5
+          env:
+            - name: RALPH_APP_DIR
+              value: "/app/.ralph"
+            - name: RALPH_AUTH_FILE
+              value: "/var/run/ralph/auth.json"
+            - name: RALPH_BACKENDS__DATABASE__ES__CLIENT_OPTIONS__ca_certs
+              value: "/usr/local/share/ca-certificates/es-cluster.pem"
```

Comment:
As mentioned yesterday, this can be improved thanks to a `ConfigMap` (looping over a `values.yml` file configuration) injected in the environment.